test/e2e/predictor/test_huggingface_vllm_openvino.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2024 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+from kubernetes import client
+from kubernetes.client import V1ResourceRequirements
+
+from kserve import (
+    V1beta1PredictorSpec,
+    V1beta1ModelSpec,
+    V1beta1ModelFormat,
+    V1beta1InferenceService,
+    V1beta1InferenceServiceSpec,
+    KServeClient,
+)
+from kserve.constants import constants
+from ..common.utils import KSERVE_TEST_NAMESPACE, generate, predict_isvc
+from .test_output import huggingface_text_embedding_expected_output
+
+
+@pytest.mark.llm
+def test_huggingface_vllm_openvino_openai_chat_completions():
+    service_name = "hf-opt-125m-chat"
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        model=V1beta1ModelSpec(
+            model_format=V1beta1ModelFormat(
+                name="huggingface",
+            ),
+            image="quay.io/ayush_sawant/huggingface_server-cpu-openvino:v0.14.0",
```

Comment:
We can build the image in CI. https://github.com/kserve/kserve/blob/master/test/scripts/gh-actions/build-server-runtimes.sh

Reply:
I have made this change, can you please review if everything else regarding CI run is correct?

Reply:
The image needs to be downloaded here https://github.com/kserve/kserve/blob/edcd1ef72c772b222c94727882bdbcdd4ea29c1d/.github/workflows/e2e-test.yml#L914

Reply:
Yeah the path is same for cpu-openvino image as well https://github.com/kserve/kserve/blob/71628a4d3e0499f9365345d42afb269c17a956ab/test/scripts/gh-actions/build-server-runtimes.sh#L83
 so it be be download here properly right?

Reply:
Here the image is built and uploaded to the github artifact. In the e2e test job you have to download the artifact and load the image